Makefile

```diff
@@ -1,6 +1,9 @@
 lint:
 	golangci-lint run
 
+format:
+	gofmt -w .
```

Comment:
I've seen some places (including GitHub Copilot) suggest `gofmt -s -w .` (with `-s`).

That supposedly simplifies code (whatever `simplify` means). I'd like to hear the opinion of someone with more Go experience than me on the convenience of using `-s`.

Reply:
I think simplify does thinks like converting `var i int = 1` to `i := 1`.

We can add it. I even prefer gofumpt, that does eve more things... but that's another topic.